### DISCUSSION TITLE
Sandbox project name change proposal

### DISCUSSION THREAD

**Discussion by scottrigby** [open-gitops/project#91] [General]
## Name change proposal

Currently the "GitOps Working Group" name is shared between both the sandbox project, as well as the limited-time working group under App Delivery SIG. The purpose of the working group is to solidify the sandbox project, which will live on.

Suggestion:
1. Working group: keep "GitOps Working Group" as-is
2. Sandbox project: change to something else. For example, ~"GitOps Initiative"~ "OpenGitOps" 

## Status

Per governance:
- [x] this idea was raised in the [Feb 11, 2021 GitOps Working Group meeting](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.w91ueqcmi7nz) where it was met with enthusiastic support
    - Note it was raised again in context of a proposed logo (in progress) https://github.com/gitops-working-group/gitops-working-group/discussions/63#discussioncomment-418201
- [x] Starting discussion is the next step (also discussed in WG meetings and CNCF #wg-gitops slack)
- [x] Try for consensus (sent last day reminder to Slack and cncf-sig-app-delivery mailing list)
~- [ ] Vote if needed by **5PM EST Friday, March 19 2021**~

## Also see

- WG definition: https://github.com/cncf/toc/blob/main/sigs/cncf-sigs.md
- Initial sandbox: issue https://github.com/gitops-working-group/gitops-working-group/issues/32

**Comment by tonit** [comment]
All up for this. 
Since the sandbox project exists now (does it? I spoke with Katelin from the CNCF the other day and i think she mentioned that it does.. not 100% sure) the renaming should happen rather sooner then later. A late name-change just confuses people. And now stuff like logo, website etc. is being created..

**Reply by tonit** [reply]
Looks like thats the CNCF Sandbox Ticket: https://github.com/gitops-working-group/gitops-working-group/issues/32

**Reply by scottrigby** [reply]
I clarified the initial discussion description above. The proposal is to keep the working group as-is, and change the _project_ name to something that clarifies the goal of the project itself. I'll add some more updated ideas below.

**Comment by roberthstrand** [comment]
In my opinion, changing the name of the sandbox project makes perfect sense. I feel that the "working group" doesn't work in a long term sense.

**Comment by tonit** [comment]
What are the alternatives for "GitOps Initiative" ? I personally really already accustomed with this one.

Lets me just come up with alternatives to have something to discuss on:
- "GitOps Project" 
- "GitOps Group"
- "GitOps Alliance"
- "GitOps Community"
- "GitOps 21" (as in "formed in 2021" but also a reference to the 21st century)
- "GitOps Peer Group"
- "Friends of GitOps"
- "GitOps Rocks" (also i heard gitops.rocks domain was available - not anymore.)
- "GitOps Rockstars" (Who doesn't want to participate in THAT project?)
-  "GitOps Team" (gitops.team is available)

**Reply by roberthstrand** [reply]
Do we need a different name? I think that _initiative_ is pretty descriptive, although I always wanted to be a part of an alliance 😁

**Reply by tonit** [reply]
Initiative would be a new name. It has not been decided.
It is also my favorite atm.

**Reply by roberthstrand** [reply]
I meant, a different than the one suggestion we already had. I like initiative, it fits.

**Comment by bboreham** [comment]
"OpenGitOps"

(by analogy with OpenMetrics, OpenTracing)

**Reply by tonit** [reply]
@cdavisafc how shall we proceed here? We Agreed on timeboxing this. Who is able to do that?

**Reply by scottrigby** [reply]
@tonit  this week I'm planning to organize this and other goals into GitHub projects, so we can prioritize tasks, organize into milestones, etc.

**Comment by tonit** [comment]
"Open GitOps" or "OpenGitOps" - OGO for short, seems not super fancy but could open some doors. 
"Open" is a prefix that does not conflict with things like setting standards, providing good defaults, best practices with a "this is not a vendor lock-in thing but exactly the opposite" theme.

What do others think?

**Reply by scottrigby** [reply]
I like this personally, let's get some more people's thoughts.

One fun note is someone made this empty org https://github.com/opengitops about 2 weeks ago (the .com and .net were also registered on that day). Do we know who?

Just for kicks I registered .io, and .org (similar to opencontainers.org). If we go with this I'll transfer them to CNCF project ownership.

**Reply by todaywasawesome** [reply]
Yeah we need to track down who snagged them!

**Reply by scottrigby** [reply]
@todaywasawesome added to project board: https://github.com/orgs/gitops-working-group/projects/6#card-57308590

**Comment by dj1k** [comment]
I like @bboreham's suggestion for OpenGitOps.

**Comment by cdavisafc** [comment]
Thank you all for the discussion. 

I believe it is very important that we make this name change. I firmly believe that GitOps will come to be the dominant operational model for cloud native and that it warrants a lasting home. The CNCF TOC agreed and that is why we are a sandbox project. The WG is in service to this project.

So I move that we adopt the project name OpenGitOps (great idea @bboreham!).

**Reply by dj1k** [reply]
Seconded.

**Comment by todaywasawesome** [comment]
Excellent work everyone I think OpenGitOps has a good ring to it and we already have a good consensus going. Great idea @bboreham  thanks @scottrigby @cdavisafc for driving it forward.

**Comment by scottrigby** [comment]
Ok everyone, it's that time 🕔  There has been no opposition. We have reached consensus on **OpenGitOps** 🎉 🙌 

For transparency, I've checked:
- Comments and emoji reactions on this issue 🤩 
- CNCF Slack `#wg-gitops`
- cncf-sig-app-delivery mailing list

As noted in Slack and the mailing list, this is a vendor-neutral, community movement that everyone has a chance to participate in. We're all very excited to keep this momentum going. Thanks to all who participated! 🚀

**Reply by cdavisafc** [reply]
Thank you for shepherding this @scottrigby!

I am in the process of updating the governance document to issue a PR with WG co-chairs. I will also include verbiage that reflects this change. Will issue a PR shortly.

**Reply by scottrigby** [reply]
Cross-referencing this clear response by @cdavisafc in the [cncf-sig-app-delivery mailing list thread](https://lists.cncf.io/g/cncf-sig-app-delivery/topic/gitops_wg_sandbox_project/81460182?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,0,81460182):

>While the working group was the initial idea, we quickly came to the conclusion that we needed a project that would outlive the WG. So it is that the WG supports the project, not the other way around. In fact, if you have a listen to the [sandbox review of Jan 26](https://www.youtube.com/watch?v=C1rPSU07lQQ), where the GitOps sandbox project was approved by the TOC, the members expressed some concern with having a sandbox project with "working group" in the name - they ultimately suggested that when the time was right, that concern would be addressed. That is exactly what we are doing right now. As Scott summarized here, we have general consensus that this renaming reduces confusion.
